### PR TITLE
Conlt 224/rate plans pdf

### PR DESCRIPTION
**Pull request by ShrabaneeMishra**
As part of [https://aussiecommerce.atlassian.net/browse/CONLT-224](https://aussiecommerce.atlassian.net/browse/CONLT-224)

An API to download rateplans list, which are sent along with the request body.

Sample o/p:

[LuxuryEscapes-Rate-Plans-00128000005c1SvAAI.pdf](https://github.com/lux-group/svc-pdf/files/8340937/LuxuryEscapes-Rate-Plans-00128000005c1SvAAI.pdf)

### CODE REVIEW

**Comment by daniel-kho-NZ** [comment]
I see a lot of `var` keyword getting used. Could we use `let` or `const` to keep ES6 standard?

**Review by daniel-kho-NZ — APPROVED** [review]

File: dist/pdf/TaxInvoice/TableBodyComponent.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  var showGST = _ref.showGST,
+      items = _ref.items,
+      order_currency = _ref.order_currency;
+
+  var totalAmount = function totalAmount(total_price) {
+    return (0, _formatCurrency.formatDynamicAmount)(total_price, order_currency);
+  };
+
+  // don't show gst for international order
+  console.log('showGST === :: ' + showGST);
```
Comment (RyanKeepRunning) [inline dist/pdf/TaxInvoice/TableBodyComponent.js:29]: Are these console.logs required :D

**Review by RyanKeepRunning — APPROVED** [review]

File: dist/pdf/TaxInvoice/TableBodyComponent.js (hunk 1 above)
Comment (ShrabaneeMishra) [reply dist/pdf/TaxInvoice/TableBodyComponent.js:29]: Nope. Will remove it :) 
Thanks

**Comment by ShrabaneeMishra** [comment]
@daniel-kho-NZ  the `var`s are in `dist` folder which is auto generated 👍 
@RyanKeepRunning I was thinking of adding the folder in `.gitignore`.  Thoughts?

**Pull request merged**